warehouse/internal/repo/staging.go

```diff
@@ -251,22 +251,43 @@ func (repo *StagingFiles) GetByID(ctx context.Context, ID int64) (model.StagingF
 	return *entries[0], err
 }
 
-// GetSchemaByID returns staging file schema field the given ID.
-func (repo *StagingFiles) GetSchemaByID(ctx context.Context, ID int64) (jsonstd.RawMessage, error) {
-	query := `SELECT schema FROM ` + stagingTableName + ` WHERE id = $1`
+// GetSchemasByIDs returns staging file schemas for the given IDs.
+func (repo *StagingFiles) GetSchemasByIDs(ctx context.Context, ids []int64) ([]model.Schema, error) {
+	time.Sleep(time.Second * 5)
```

Comment:
How come you added this? What behaviour were you trying to test?

```suggestion
```

Reply:
I thought I removed this. I was trying to verify user cancel the request.